userspace/libsinsp/plugin.cpp

```diff
@@ -668,10 +658,34 @@ void sinsp_plugin::resolve_dylib_field_arg(Json::Value root, filtercheck_field_i
 
 bool sinsp_plugin::resolve_dylib_symbols(std::string &errstr)
 {
+
+	if ((*(void **) (&(m_api.get_required_api_version)) = getsym("plugin_get_required_api_version", errstr)) == NULL)
+	{
+		return false;
+	}
+
+	// The required api version was already checked in
+	// create_plugin to be valid and compatible. This just saves it for info/debugging.
```

Comment:
This comment sounds strange to me probably have to update it since we moved the function